src/main/java/Duke/DeleteCommand.java

```diff
@@ -0,0 +1,40 @@
+package Duke;
+
+/**
+ * This class encapsulates the command to delete a task.
+ * It is triggered by the parser and uses the TaskManager, Ui, and Storage.
+ */
+public class DeleteCommand implements ICommand {
+
+    String taskIndex;
+
+    /**
+     * Constructor for the command.
+     *
```

Comment:
Just something very minor, but it may look nicer if you remove this empty line to standardise it with the javadocs for your other classes' constructors?